mdtools.py

```diff
@@ -26,17 +34,36 @@ def grompp(
     if grompp_options is None:
         grompp_options = []
     output_files = {"-o": tpr_file, "-po": mdp_output_file}
-    prep = gmx.commandline_operation(
-        executable="gmx",
-        arguments=["grompp"] + grompp_options,
-        input_files=input_files,
-        output_files=output_files,
-    )
-    prep.run()
-    output = str(prep.output.erroroutput.result()).strip()
+    if use_api:
+        prep = gmx.commandline_operation(
+            executable="gmx",
+            arguments=["grompp"] + grompp_options,
+            input_files=input_files,
+            output_files=output_files,
+        )
+        prep.run()
+        output = str(prep.output.erroroutput.result()).strip()
+    else:
+        infiles = ' '.join([k + ' ' + v for k, v in input_files.items()])
+        outfiles = ' '.join([k + ' ' + v for k, v in output_files.items()])
+        gmx = "mpiexec -n 1 gmx_mpi" if gmx_executable == "gmx_mpi" else "gmx"
```

Comment:
So, mpiexec in my cluster needs to be srun... maybe there should be an input variable for this in config.json